src/Neo.VM/ScriptBuilder.cs

```diff
@@ -126,7 +126,8 @@ public ScriptBuilder EmitPush(bool value)
         /// <returns>A reference to this instance after the emit operation has completed.</returns>
         public ScriptBuilder EmitPush(ReadOnlySpan<byte> data)
         {
-            if (data == null)
+            ReadOnlySpan<byte> nullData = null;
```

Comment:
What is this?

Reply:
Because `if (data == null)` will fail the format check in .net 9.